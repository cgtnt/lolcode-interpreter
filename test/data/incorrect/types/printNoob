HAI

HOW IZ I ADD YR A AN YR B
  SUM OF A AN B
  GTFO
IF U SAY SO

VISIBLE I IZ ADD YR 2 AN YR 3 MKAY!

KTHXBYE
