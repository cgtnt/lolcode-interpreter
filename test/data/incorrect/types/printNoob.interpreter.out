Line 8: Cannot cast NOOB to YARN
